### ISSUE TITLE
WCAG 3.3.2: Labels or Instructions

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5165]
**Level:** A
**Principle:** Understandable
**Result:** *pending*

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html)

### Audit Checklist
- [ ] Review previous Audit Result
- [ ] Review work identified in previous audit and list open issues.
- [ ] Check for any other relevant open issues.
- [ ] Retest open issues, are these issues still present?
- [ ] Test for any new issues that may have been introduced.
- [ ] Determine compliance status, add this to issue name.
- [ ] Add any new issues to the Accessibility project for triage.

### Previous audit
 - link previous audit and summarise what was find and the outcome.

### Automated Tests
no violations from 
- form-field-multiple-labels — Form field must not have multiple label elements
- label-title-only — Form elements should have a visible label

A handful from
- select-name — Select element must have an accessible name
- label — Form elements must have labels

 ### Progress since last audit

 ### Audit Notes

 ### Work arising
